client/onboarding/utils.ts

```diff
@@ -42,6 +42,18 @@ export const getBusinessTypes = (): Country[] => {
 	);
 };
 
+export const getMccFromIndustry = (): string | undefined => {
+	const industry = wcSettings.admin.onboarding.profile.industry?.[ 0 ];
+	if ( ! industry ) {
+		return undefined;
+	}
+
+	const industryToMcc =
+		wcpaySettings?.onboardingFieldsData?.industry_to_mcc || {};
+
+	return industryToMcc[ industry ];
```

Comment:
If you are inclined toward my previous suggestion, this will need to change to:

```suggestion
	return industryToMcc[ industry ] || null;
```

Reply:
This is not needed anymore. See https://github.com/Automattic/woocommerce-payments/pull/8463#discussion_r1540723281.